packages/react-components/react-tags-preview/src/components/InteractionTagPrimary/useInteractionTagPrimary.ts

```diff
@@ -51,6 +51,7 @@ export const useInteractionTagPrimary_unstable = (
       getNativeElementProps('button', {
         ref,
         disabled,
+        id: interactionTagPrimaryId,
         ...props,
```

Comment:
`props.id` can override the id provided by context, it would be nice to have it documented that if the user provides a custom `id` a custom `aria-labelledby` will also be required

Reply:
Added as 
<img width="1168" alt="image" src="https://github.com/microsoft/fluentui/assets/28751745/75e827ef-6128-4f5a-909f-6b91f458c948">